### PR TITLE
New bits in ScummVM doc

### PR DESCRIPTION
**Pull request by mrmatteastwood**
Hello team! I added a bit of info to the ScummVM doc. First, in the MIDI/FluidSynth section, I pointed to the very good FatBoy soundfont as an alternative to the Roland SC-55 soundfont. Secondly, in "compatibility", I explained how to run Westwood games (Legend of Kyrandia and some others) using the correct kyra.dat for whatever version of ScummVM one is using.

### CODE REVIEW

**Comment by mrmatteastwood** [comment]
Cripes, I just realized that my WIP version of the DOSBox Pure documentation I started a few months ago also went into this pull request. I don't know GitHub well enough to understand how to make a pull request only for the changes I made today. Please ignore the dosbox-pure.md, I need to get back together with @schellingb to finish that up.

**Comment by fpscan** [comment]
Let me know when you think it's ready to merge and you may want to split the changes, you can create branches based repo. So make sure main repo is up to date then create new branch then go to back main repo and create another branch from there. So you can work on multiple changes.

**Comment by mrmatteastwood** [comment]
Thanks @fpscan! I think I got it. I made a branch inside my own repo for the DBP doc, and I deleted the file from the master. So now only my changes on scummvm.md should be left and everything should be good to merge.

**Review by fpscan — APPROVED** [review]

File: docs/library/scummvm.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ Some games only contain music in the form of MIDI data. By default, ScummVM will
 
 - Press the 'SoundFont:' button and navigate to the RetroArch `system` directory. Enter the `scummvm/extra/` folder. (NB: If the steps of this guide have been followed in order, the `scummvm/extra/` folder will be selected automatically upon pressing the 'SoundFont:' button)
 
-- Select the file `Roland_SC-55.sf2` and press the 'Choose' button.
+- Select the file `Roland_SC-55.sf2` and press the 'Choose' button. Another good soundfont is the [FatBoy SoundFont](https://fatboy.site/), you can download that and place it in the same folder (`scummvm/extra/`) to be able to select it.
```
[the remark this answers is not in the document]
Comment (mrmatteastwood) [reply docs/library/scummvm.md:131]: Yeah, I noticed the same thing and sent them an e-mail a few days ago, but they haven't responded. I found a copy on archive.org, but it was an outdated version. So I am hosting the latest on my own website now and I'll add the link to the doc. New pull request coming up.

**Pull request merged**